client/promises.js

```diff
@@ -72,33 +120,37 @@ ajaxPromise()
 //   })
 // };
 
-// pie();
+const pie = () => {
+  return axios.get('http://127.0.0.1:3030/pies')
+  .then(data => data.data)
+  .catch(err => err);
+}
 
-// const pieCounter = () => {
-//   // FILL ME IN
-// };
+const pieCounter = () => {
+  // FILL ME IN
+};
 
 // pieCounter();
 
-// ===================================================================== //
+// ========================================================================================================= //
 
-/* 
-*  The below is some extra practice using promises with some of the fs methods you worked with on a-sync-swim.
+/*
+*  The content below is some extra practice using promises with some of the fs methods you worked with on a-sync-swim.
```

Comment:
The junior phase no longer uses async-swim -- in case of future changes again, I would just say "server-side concepts sprint" to generalize it.  Also this portion doesn't use fs until the server side so that might be confusing 🤔